marklogic-data-hub/build.gradle

```diff
@@ -32,6 +32,7 @@ dependencies {
     compile 'com.marklogic:marklogic-data-movement-components:1.0'
     compile 'commons-io:commons-io:2.4'
     compile 'org.apache.commons:commons-text:1.1'
+    compile 'org.json:json:20180813'
```

Comment:
you should not need to include this.  I'll find where you imported it and suggest an alternative